.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.6.2
```

Comment:
See my comment in the other PR -- remove from the PR (add to `.gitignore` if desired)